### ISSUE TITLE
[Manage Organization] Form does not use proper element tag and some inputs are missing inline error

### ISSUE THREAD

**Issue by johanna-skylight** [CDCgov/prime-simplereport#5814]
## Area or Component

Settings Page -> Manage Organization Tab

## Quick Summary

The form to update the organization information is not set with a form tag and the organization name field does not display inline errors (just the toaster)

### How to Replicate

Try to submit an empty organization name and verify no inline error displays.
Inspect the DOM and verify the form is currently using a div as a wrapper.

## Accessibility Violation

The required field is not identified by any of the following means: its label or form-level instructions or an error message. https://docs.deque.com/issue-help/1.0.0/en/form-error-required-unclear

## Action Requested

Swap the div wrapper with a form tag.
Add inline error to the organization name input field.

As a stretch migrate this form to use react-hook-forms

## Screenshots (optional)
![Screenshot 2023-05-16 at 12 11 39 PM](https://github.com/CDCgov/prime-simplereport/assets/103958711/8d5df76b-ce59-4063-bfe0-fa316293aa91)
